testing/test_remote.py

```diff
@@ -220,6 +222,7 @@ def test_remote_env_vars(testdir):
         """
         import os
         def test():
+            assert len(os.environ['PYTEST_XDIST_SESSIONUUID']) == 32
```

Comment:
You could go for testing that the 32-char string is not just any string, but in uuid hex format. Same above in `acceptance_test.py`

Reply:
Ehm I think it's fine. If it's exactly 32 chars and we're supposed to be looking at a uuid then we can be reasonably sure that everything is fine.